### PR TITLE
Minor typo & formatting fixes

### PR DESCRIPTION
**Pull request by mkubala**

### CODE REVIEW

File: sections/bottom_types.adoc [hunk 1]
```diff
@@ -62,7 +62,7 @@ Int
 TypeRef(TypeSymbol(final abstract class Int extends AnyVal))
 ```
 
-The verbose option adds a bit more information here, now we know that `Int` is an `AnyVal` - this is a special class representing value types - which cannot hold `Null`. If we look into <a href="https://github.com/scala/scala/blob/v2.10.3/src/library/scala/AnyVal.scala">AnyVal's sources</a>, we'll find:
```
Comment (vendethiel) [inline sections/bottom_types.adoc:65]: I think you should use the `[caption](link)` form

Comment (mkubala) [inline sections/bottom_types.adoc:65]: `[caption](link)` is a correct form for markdown-based documents, but this project uses the [Asciidoctor](http://asciidoctor.org/) with it's `link[caption]` form (http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#links).
Anyway, thank you for being watchful :+1:

Comment (vendethiel) [inline sections/bottom_types.adoc:65]: Oh, that explains why some markdown links weren't showing in markdown form! thanks

**Comment by ktoso** [comment]
thanks :)

**Pull request merged**